src/effects/Tween.js

```diff
@@ -14,6 +16,8 @@ Tween.prototype = {
 	constructor: Tween,
 	init: function( elem, options, prop, end, easing, unit ) {
 		this.elem = elem;
+		this.styleElem = elem.nodeType === 1 && ( jQuery.cssHooks[ prop ] ||
```

Comment:
What's the reason for this change? We're now unit-testing `jQuery.Tween`, in particular the behavior of its properties so it's kind of a public API now. I'd rather not expose internal properties on Tween objects.

Reply:
Mainly I was trying to avoid doing this calculation each time and just do it once in the init. It's no problem to move it back though.